client/coral-framework/reducers/auth.js

```diff
@@ -14,12 +15,18 @@ const purge = user => {
 
 export default function auth(state = initialState, action) {
   switch (action.type) {
+    case actions.SET_AUTH_TOKEN:
+      return {
+        ...state,
+        token: action.token || null,
```

Comment:
Normally logic is not allowed in the reducer. This should be break down into 2 actions or send the value when it's dispatched - This is standard, but I'm fine either way.